src/clj/rems/css/styles.clj

```diff
@@ -638,8 +638,9 @@
              :margin-bottom (u/px 16)}]
    [:div.form-control {:height :auto
                        :white-space "pre-wrap"
-                       :border-color "rgba(206, 212, 218, 0.2)" ; "#ced4da"
-                       :background-color "rgba(0, 0, 0, 0.01)"}
+                       ;; A copy of bootstrap's .form-control:disabled
+                       :background-color "#e9ecef"
+                       :opacity 1}
```

Comment:
I'm not sure this is actually so useful change as I thought. We are overriding a customization with another customization, only now we try to manually track Bootstrap styling. If we could remove some of our customizations and directly use Bootstrap classes, that'd work for me, i.e. simplifying stuff. Now we are trading one look for another with no clear use case. 

Is this trying to solve some contrast issue? Doesn't this reduce the contrast?

Reply:
Yeah we didn't get rid of custom styles unfortunately. I would like to get bootstrap's `.form-control:disabled` style to fire, but the `:disabled` selector doesn't work on divs (even though I can mark the div as disabled). We need a div.form-control instead of an input.form-control because we want to have arbitrary html inside (e.g. lists).

We are trading one look for another, but we gain:
- consistency (see the table, and we might have other disabled inputs somewhere too)
- better contrast (between the white background and the input box)
- less custom colours

I'm not sure it's a slam dunk but it might be a net improvement.